dispute.go

```diff
@@ -0,0 +1,65 @@
+package braintree
+
+import "time"
+
+const (
+
+	// Dispute kinds
+	DisputeKindChargeback     = "chargeback"
+	DisputeKindPreArbitration = "pre_arbitration"
+	DisputeKindRetrieval      = "retrieval"
+
+	// Dispute reasons
+	DisputeReasonCancelledRecurringTransaction = "cancelled_recurring_transaction"
+	DisputeReasonCreditNotProcessed            = "credit_not_processed"
+	DisputeReasonDuplicate                     = "duplicate"
+	DisputeReasonFraud                         = "fraud"
+	DisputeReasonGeneral                       = "general"
+	DisputeReasonInvalidAccount                = "invalid_account"
+	DisputeReasonNotRecognized                 = "not_recognized"
+	DisputeReasonProductNotReceived            = "product_not_received"
+	DisputeReasonProductUnsatisfactory         = "product_unsatisfactory"
+	DisputeReasonTransactionAmountDiffers      = "transaction_amount_differs"
+
+	// Dispute statuses
+	DisputeStatusAccepted = "accepted"
+	DisputeStatusDisputed = "disputed"
+	DisputeStatusExpired  = "expired"
+	DisputeStatusOpen     = "open"
+	DisputeStatusLost     = "lost"
+	DisputeStatusWon      = "won"
+)
```

Comment:
For each of these could you add a type, so that they are not just strings, and make the corresponding fields on the structs those types? Example:

https://github.com/lionelbarrow/braintree-go/blob/273799669d13c0589383b83d45239c5b5819528a/transaction.go#L10-L17

Reply:
Agreed.